0.4/Dockerfile

```diff
@@ -32,10 +32,13 @@ ENV JANUS_VERSION=${JANUS_VERSION} \
     janusgraph.storage.directory=/var/lib/janusgraph/data \
     gremlinserver.graph=/etc/opt/janusgraph/janusgraph.properties \
     gremlinserver.threadPoolWorker=1 \
-    gremlinserver.gremlinPool=8
+    gremlinserver.gremlinPool=8 \
+    DEBIAN_FRONTEND=noninteractive
 
 RUN groupadd -r janusgraph --gid=999 && \
     useradd -r -g janusgraph --uid=999 janusgraph && \
+    apt-get update -yqq && \
```

Comment:
(nitpick) Do we really need to make this completely quiet? Wouldn't a bit of output be helpful, especially if this fails for some reason?

Reply:
Done. Removed quiet flags.

> You need [Kerberos for Solr](https://docs.janusgraph.org/index-backend/solr/#kerberos-configuration), right? A comment directly in the `Dockerfile` could also be a good idea here to describe why this dependency is needed for future contributors.
> If a backend we ship needs such a dependency for a certain functionality, then I think it's OK to add it.
> 
> I added a few minor comments. Apart from that, please don't edit the `Dockerfile` by hand as it's generated automatically from the `bulid/Dockerfile-openjdk8.template` as indicated in its header. Instead, modify the template and then run the `update.sh` script to generate the `Dockerfiles` for the different versions.

We need kerberos support for any secured backend storage.(In our case, HBase).

Thanks for heads up !! made changes via docker template.

Reply:
Thanks! The changes in the template now look good to me in general, but could you please execute `update.sh`? This will update this `Dockerfile` and the one in `0.3`.

Reply:
done